### DISCUSSION TITLE
Paperless NGX 2.13.5

### DISCUSSION THREAD

**Discussion by Beo666** [community-scripts/ProxmoxVE#255] [Q&A] [answered]
Hello Community,
I've updated my paperless installation yesterday to 2.13.5 and now the KI classifieing won't work any more. I get the mesagge in the logs: 

> [2024-11-14 16:36:38,864] [ERROR] [paperless.api] System status detected a possible problem while loading the classifier: Classifier file does not exist (yet). Re-training may be pending.
> Traceback (most recent call last):
>   File "/opt/paperless/src/documents/views.py", line 2074, in get
>     raise FileNotFoundError(classifier_error)
> FileNotFoundError: Classifier file does not exist (yet). Re-training may be pending.

Perhaps someone can help me?

Best regards
Lukas

**Comment by costa-alex** [comment, marked as answer]
Had the same issue yesterday after updating. Fixed it by running `python3 manage.py document_create_classifier` inside paperless/src folder.

**Comment by Beo666** [comment]
Thank you very much, this was the solution.
